src/mesh/MFEMMesh.C

```diff
@@ -58,6 +69,14 @@ MFEMMesh::buildDummyMooseMesh()
   getMesh().prepare_for_use();
 }
 
+void
+MFEMMesh::uniformRefinement(mfem::Mesh & mesh, int nref)
+{
+  std::cout << "Got into UniformRefinement! Will be doing " << nref << " refinements." << std::endl;
+  for (int i = 0; i < nref; ++i)
+    mesh.UniformRefinement();
```

Comment:
Currently no tests check the refinement actually works - worth adding a new unit test or regression test to check this (starting from a small mesh to keep runtimes low!). 

The added options in the Diffusion test are set to zero, which are the default values here, so the crucial refinement steps are not tested.

Reply:
Yeah! That's a good idea. I'll add a test

Reply:
Done!